src/SizeBench.PathLocators/IBinaryLocator.cs

```diff
@@ -3,4 +3,5 @@
 public interface IBinaryLocator
 {
     bool TryInferBinaryPathFromPDBPath(string pdbPath, out string binaryPath);
-}
\ No newline at end of file
+    bool TryInferPDBPathFromBinaryPath(string binaryPath, out string pdbPath);
```

Comment:
Thank you for adding this, I've meant to do it for a long time since so many folks want to enter the binary and find the PDB, not vice-versa, in the UI.